app/src/androidTest/java/com/bottlerocketstudios/mapsdemo/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.bottlerocketstudios.mapsdemo
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        assertEquals("com.example.android_maps_demo", appContext.packageName)
```

Comment:
@Josh-Gebbeken nit: Need to refactor this, likely to `com.bottlerocketstudios.mapsdemo`

```suggestion
        assertEquals("com.bottlerocketstudios.mapsdemo", appContext.packageName)
```